pkg/resolver/search.go

```diff
@@ -77,6 +98,9 @@ func (s *SearchResult) Count() (int, error) {
 func (s *SearchResult) Items() ([]map[string]interface{}, error) {
 	s.wg.Add(1)
 	defer s.wg.Done()
+	if !s.isManagedHub() { // if current hub is not part of managedHub filter, stop search
+		return nil, nil
```

Comment:
There's a slight difference with returning nil vs. an empty []map[string]interface{}.  I think an empty list is slightly better and safer.